postgres/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ***Added***:
 
+* Establish a dedicated main db connection with heath check ([#15962](https://github.com/DataDog/integrations-core/pull/15962))
```

Comment:
```suggestion
* Establish a dedicated main db connection to prevent the main thread db from closing prematurely ([#15962](https://github.com/DataDog/integrations-core/pull/15962))
```